Doc/library/datetime.rst

```diff
@@ -84,6 +84,10 @@ The :mod:`datetime` module exports the following constants:
    The largest year number allowed in a :class:`date` or :class:`.datetime` object.
    :const:`MAXYEAR` is ``9999``.
 
+.. attribute:: UTC
```

Comment:
I think this is maybe supposed to be `data` like the ones above it?

Reply:
I looked at other modules, and it seemed that attribute would fit better here since `UTC` is not technically a constant.

Reply:
OK, that's fine.